Lib/test/test_platform.py

```diff
@@ -275,6 +275,26 @@ def test_libc_ver(self):
         self.assertEqual(platform.libc_ver(support.TESTFN),
                          ('glibc', '1.23.4'))
 
+    @support.cpython_only
+    def test__comparable_version(self):
+        from platform import _comparable_version as V
```

Comment:
What do you think of also adding tests checking directly the result of the function? Compare to a list.

Reply:
This would expose too much implementation details. For example the initial versions converted '1.2.3' to `[(100, 1), (100, 2), (100, 3)]`, but now it converts to `[100, 1, 100, 2, 100, 3]`. I thought also for converting to `[1, 2, 3]` and using negative numbers for pre-release versions and `float('inf')` for "pl". If this function be public, I would return a tuple, but a list is enough in this case. Of course magic numbers like 100 are arbitrary.

Reply:
Ok, it's up to you.